front/components/vaults/ConnectorCreatedModal.tsx

```diff
@@ -0,0 +1,103 @@
+import { Icon, Modal, Page, SparklesIcon } from "@dust-tt/sparkle";
+import type {
+  ConnectorStatusDetails,
+  DataSourceType,
+  LightWorkspaceType,
+} from "@dust-tt/types";
+
+import ConnectorSyncingChip from "@app/components/data_source/DataSourceSyncChip";
+
+type DataSourceViewSelectionModalProps = {
+  dataSource: DataSourceType & ConnectorStatusDetails;
+  isOpen: boolean;
+  onClose: (shouldRefresh: boolean) => void;
+  owner: LightWorkspaceType;
+};
+
+export const ConnectorCreatedModal = ({
+  isOpen,
+  onClose,
+  dataSource,
+  owner,
+}: DataSourceViewSelectionModalProps) => {
+  // const [showVaultCreationModal, setShowVaultCreationModal] = useState(false);
+  // const [defaultVault, setDefaultVault] = useState<VaultType | null>(null);
+  // const { vaults, isVaultsLoading } = useVaultsAsAdmin({
+  //   workspaceId: owner.sId,
+  // });
```

Comment:
✂️

Reply:
Well there's a bunch of commented code here?

Reply:
that's for the VaultSelector  we want to include next .. can get rid of it for now